docs/chapter-5.mdx

```diff
@@ -1,90 +1,205 @@
+
+@Component({/* ... */})
+export class AppComponent implements OnInit {
+  ngOnInit() {
+    console.log(addCategory({category:{ name: "Food" }}));
+  }
+}
+```
+
+In the console, we will see the following:
+
+```js
+{category: {name: 'Food', type: '[Category List] Add Category'}}
+```
+
+So essentially, `createAction` provided us with an easy way of creating a single action. `addCategory` in our case is an `ActionCreator`, a function which produces an action object whenever called, and `props` explained what argument that `ActionCreator` function expects.
+
+#### Using `createActionGroup`
+
+As we just saw, creating an action is pretty straightforward.
+
+However, when dealing with multiple actions, this can led to a more verbose code and prone to typos:
+
+```ts title="state/actions.ts"
+export const addCategory = createAction(
+  "[Category List] Add Category",
+  props<{ category: Category }>()
+);
+
+export const addAnotherCategory = createAction(
+  //  👇 Did you notice the typo?
+  "[Catgory List] Add Another Category",
+  props<{ category: Category }>()
+);
+```
+
+With those issues in mind, NgRx published a new function that allows us to create multiple actions at one: the `createActionGroup` function.
+
+Its usage is fairly simple and is based on two parameters:
+
+- The source of the action, in our case `Category List`
+- A serie of properties symbolizing our actions and their parameters
+
+Let's go back to the previous example to see what is changing:
+
+```ts title="state/actions.ts"
+export const categoryActions = createActionGroup({
+  source: 'Category List',
+  events: {
+    'Add Category': props<{ category: Category }>(),
+    'Add Another Category': props<{ category: Category }>(),
+  }
+})
+```
+
+By doing so, all our related actions will be grouped together and we still can create them by calling the `categoryActions` variable that automatically generates you the code to create them:
+
+```ts
+categoryActions.addCategory({ category: 'Food' });
+categoryActions.addAnotherCategory({ category: 'Food' });
+```
+
+## Homework
+
+Yes, you've read it correctly: we have learned how to write some basic code in NgRx, so it is time for some homework!
+
+:::info
+
+We **strongly** recommand you not to move on to the next chapter without adding the homework code as we will be using that code in the next chapters
+
+:::
+
+1. **Create an action for deleting a category**  
+   It should receive a string with the name of the category, and in the next chapter we will use that code to write the actual logic for deleting the category.
+2. **Create an action for updating a category**  
+   It must receive a `Category` object (`{name: string}`), and again, we will write the code to update the category in the next chapter
+
+> For this homework, assume categories cannot have duplicate names. We will deal with this problem later.
+
+:::note
+
+You will find solution code for all the homeworks in the end of the chapters

```

Comment:
nit: let's write "at the end" instead of "in the end"

Reply:
Thanks for the nitpicking, I definitely should improve my English 😅